cmd/batchStorageSync/main.go

```diff
@@ -21,31 +20,36 @@ import (
 	"github.com/iryonetwork/wwm/utils"
 )
 
-var (
-	boltFilepath  string = "/data/batchStorageSync.db"
-	storageBucket string = "batchStorageSync"
-	storageKey    string = "lastSuccessfulRun"
+const (
+	StorageBucket string = "batchStorageSync"
+	StorageKey    string = "lastSuccessfulRun"
```

Comment:
why did you change the first letter to upper case?

Reply:
I see no valid reason actually, I guess I felt like const is more important or whatever :D but since it does not need to be exported, will revert.

Reply:
changed